src/transformers/Twitter.js

```diff
@@ -7,17 +7,26 @@ export const shouldTransform = url => {
 
   return (
     ['twitter.com', 'www.twitter.com'].includes(host) &&
-    pathname.includes('/status/')
+    (pathname.includes('/status/') ||
+      ((pathname.includes('/moments/') || pathname.includes('/events/')) &&
```

Comment:
Use the `includesSomeOfArray` function instead

```js
includesSomeOfArray(pathname, ['/events/', '/moments/'])
```

Reply:
Neat! Didn't notice we had that function available :D